packages/terra-detail-view/src/DetailViewSubtitle.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  /**
+   * Text to display as a subtitle
+   **/
+  text: PropTypes.string.isRequired,
+};
+
+const DetailViewSubtitle = ({ text, ...customProps }) => {
+  const attributes = Object.assign({}, customProps);
+  attributes.className = classNames(['terra-DetailView-subtitle',
```

Comment:
Nit: Look at matching this implementation with https://github.com/cerner/terra-core/blob/master/packages/terra-slide-panel/src/SlidePanel.jsx#L64-L75